GraphQL-TSC.md

```diff
@@ -162,24 +168,44 @@ When something cannot be decided by consensus or a formal vote is required, the
 
-* A notice is sent via email that the vote will conclude in three business days, reminding those who haven't voted that they should do so. The vote will conclude at the end of this time.
-* The election results will not change if additional TSC members vote.
+A TSC member must attend two of the previous five meetings to be considered a "TSC *attending* member" and count towards a voting quorum. Should a TSC member miss three or more of the prior five meetings, they will no longer be counted when determining quorum. A member counts towards quorum as of attending a second meeting.
 
-#### Ensuring efficient votes
+A quorum of more than half (or two-thirds for a supermajority vote) of the TSC *attending* members must cast a ballot in order for a vote to be valid.
 
-A TSC member must attend two of the previous five meetings to be considered a "TSC voting member" and be eligible to vote. Should a TSC member miss three or more of the prior five meetings, they will no longer vote or be counted toward the total number of voting members. This is to ensure broad participation as well as to avoid voting deadlocks.
+### Voting process
 
-A member may regain their voting status by attending two meetings and may participate in votes in the second meeting.
+Because we work in a distributed environment, the voting process must account for a range of time zones and schedules. Once the threshold of a voting quorum has been met and a vote is valid, one of these two critera must be satisfied to conclude a vote:
 
-#### Resignation or removal of TSC members
+* A notice is sent via email that the vote will conclude in three business days, reminding those who haven't voted that they should do so. The vote will conclude at the end of this time.
+* The election results would not change with the addition of another TSC member's vote.
 
-A TSC member may voluntarily resign at any time before the end of their term by notifying the TSC chair. A TSC member may also be involuntarily removed by a supermajority (2/3) vote by eligible TSC voting members less any with a conflict of interest.
+Once a valid vote is concluded, the vote outcome is determined as follows:
 
-After a TSC seat vacancy the remaining TSC will collect self-nominations and vote on a replacement representative to complete the term. The nomination period should be open for at least two weeks, after which a vote shall be held.
+* For a simple majority (or supermajority), the votes in favor must exceed 50% (or 66%) of the total of, as of the time the vote is concluded:
+  * the number of TSC *attending* members.
+  * the number of members who are not considered a TSC *attending* member but did cast a vote.
+* For the Condorcet method, all votes received at the time the vote is concluded are considered.
```

Comment:
I had trouble understand this paragraph, would something like this rephrasing work:

> A simple majority requires more than 50% of the TSC attending members to vote in favor. 

I don't recall us referencing the Cordorcet method elsewhere. If this is the first/only time referencing the Condorcet method, considering in-lining the relevant parts and eliding the term altogether.

Reply:
But more than 50% of the attending members isn’t quite right. 

It’s:
- take the sum of attending members and non-attending members who voted
- Majority is based on that sum, so >50% of whatever the number in the first step.

Reply:
Cordorcet has been in here since the beginning, it's referenced two headings up - we use it for multiple-winner votes (eg elections)

Let me see if I can refine the explanation of this - I agree that it's hard to reason about

Reply:
Looks like I need more coffee, thanks guys, sorry for the noise, no other comments from me.